pkg/webui/console/views/device-add/repository/device-card/picture-placeholder.js

```diff
@@ -0,0 +1,35 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+
+const DevicePicturePlaceholder = ({ className }) => (
```

Comment:
We have this and \`[pkg/webui/assets/misc/end-device-placeholder.svg](https://github.com/TheThingsNetwork/lorawan-stack/pull/3742/files#diff-f1e3a9f3ba96756d173a30a7d8ed3d5c33b1f93807ba9c5cf5c6437123e1c4cd)\` now. I guess one of those can be removed.